### ISSUE TITLE
Update to latest plugin

### ISSUE THREAD

**Issue by alexcanessa** [AlexanderProd/gatsby-shopify-starter#82]
Not sure if is worth updating this repo, but the plugin now has a different API, and requires a `STORE_URL` and `ADMIN_PASSWORD`.

See [this commit](https://github.com/gatsbyjs/gatsby/commit/76a040d31fc956de26b07d9284713963dfa1cb07) for further details.

**Comment by AlexanderProd** [comment]
Yeah im not sure if its still worth but I guess it would still be good to update, I'll see if  I can finde some time to implement. 
Feel free to open a PR. :)
